packages/x-charts/src/ChartsYAxis/ChartsYAxis.tsx

```diff
@@ -174,6 +247,15 @@ function ChartsYAxis(inProps: ChartsYAxisProps) {
     x: positionSign * (axisWidth - labelHeight),
```

Comment:
It is not directly related to this PR, but I just noticed this x position could be replaced by
```suggestion
    x: positionSign * axisWidth,
```
and the label placement could be modified with 
```diff
- dominantBaseline: 'auto',
+ dominantBaseline: 'text-before-edge',
```
(or `text-after-edge` I never remember who is who)

Such that the axis label does not depend on text measure anymore

Reply:
Updated 👍 it doesn't allow us to get rid of the label height measurement, though.

Reply:
Yeah, good point 👍